src/Microsoft.AspNet.Mvc.Core/Formatters/JsonInputFormatter.cs

```diff
@@ -79,19 +73,17 @@ public override Task<object> ReadRequestBodyAsync([NotNull] InputFormatterContex
                 var jsonSerializer = CreateJsonSerializer();
 
                 EventHandler<Newtonsoft.Json.Serialization.ErrorEventArgs> errorHandler = null;
```

Comment:
Here ErrorEventArgs is conflicting with the type from `System.IO` namespace, so fully qualifying it here.